libs/utils/utils.js

```diff
@@ -404,7 +406,7 @@ async function decorateIcons(area, config) {
   loadIcons(domIcons, config);
 }
 
-async function decoratePlaceholders(area, config) {
+export async function decoratePlaceholders(area, config) {
```

Comment:
This should not be exported. You should use the function coming out of placeholders.js.